sdk/ai/azure-ai-projects/samples/agents/sample_agents_openapi.py

```diff
@@ -0,0 +1,85 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_agents_openapi.py
+
+DESCRIPTION:
+    This sample demonstrates how to use agent operations with the 
+    OpenAPI tool from the Azure Agents service using a synchronous client.
+    To learn more about OpenAPI specs, visit https://learn.microsoft.com/azure/search/search-get-started-portal
```

Comment:
https://learn.microsoft.com/azure/search/search-get-started-portal link seems to be incorrect